depends/packages/univalue.mk

```diff
@@ -0,0 +1,21 @@
+package=univalue
+$(package)_version=1.0.1
+$(package)_download_path=https://codeload.github.com/jgarzik/$(package)/tar.gz
```

Comment:
I think we should use the bitcoin/univalue.git repository to allow multiple people to maintain the repository (the subtree is also pointing to bitcoin/univalue.git).